### PR TITLE
Added MPLS RSVP TE KPI playbook and added messages for chipset rules

### PR DESCRIPTION
**Pull request by gvarao**

### CODE REVIEW

File: juniper_official/Linecard/fi-statistics.rule [hunk 1]
```diff
@@ -107,6 +107,7 @@ healthbot {
                     then {
                         status {
                             color green;
+                            message "Center chip fabric in error count:$cchip-fi-error-packets";
```
Comment (ConstrainedRandomVar) [inline juniper_official/Linecard/fi-statistics.rule]: Should we say that this "is normal" as well?

File: juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/*
+ * Detects MPLS LSP state changes and notifies when state change count 
+ * increases frequently.
+ * One inputs control detection
```
Comment (ConstrainedRandomVar) [inline juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule]: Please change to one input controls detection.

File: juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            /*
+             * Anomaly detection logic.
+             */
+            trigger lsp-flap-count {
+                frequency 90s;
+                term lsp-stable {
+                    when {
+                        exists "$lsp-name";
+                        increasing-at-most-by-value "$lsp-state-change-count" {
+                            value 1;
```
Comment (ConstrainedRandomVar) [inline juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule]: Should this be 0? As in we don't want the count to increase at all, right?

File: juniper_official/Protocols/Mpls/check-lsp-state-oc.rule [hunk 4]
```diff
@@ -0,0 +1,89 @@
+/*
+ * Monitors MPLS LSP state and notifies when LSP state is UP or DOWN.
+ * One inputs control detection
```
Comment (ConstrainedRandomVar) [inline juniper_official/Protocols/Mpls/check-lsp-state-oc.rule]: Change to "One input controls detection".

File: juniper_official/Protocols/Mpls/check-rsvp-neighbor-state-oc.rule [hunk 5]
```diff
@@ -0,0 +1,100 @@
+/*
+ * Monitors MPLS LSP state and notifies when LSP state is UP or DOWN.
+ * One inputs control detection
```
Comment (ConstrainedRandomVar) [inline juniper_official/Protocols/Mpls/check-rsvp-neighbor-state-oc.rule]: And here..

File: juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule (hunk 2 above)
Comment (gvarao) [reply juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule]: Changed to "one input controls detection"

File: juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule (hunk 3 above)
Comment (gvarao) [reply juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule]: Yes, One makes sense. changed

File: juniper_official/Protocols/Mpls/check-lsp-state-oc.rule (hunk 4 above)
Comment (gvarao) [reply juniper_official/Protocols/Mpls/check-lsp-state-oc.rule]: Changed to "one input controls detection"

File: juniper_official/Protocols/Mpls/check-rsvp-neighbor-state-oc.rule (hunk 5 above)
Comment (gvarao) [reply juniper_official/Protocols/Mpls/check-rsvp-neighbor-state-oc.rule]: Changed to "one input controls detection"

File: juniper_official/Linecard/fi-statistics.rule (hunk 1 above)
Comment (gvarao) [reply juniper_official/Linecard/fi-statistics.rule]: Yes, changed

**Pull request merged**